docs/nidigital/rep_caps.rst

```diff
@@ -110,4 +110,30 @@ conditional_jump_triggers
         passes a string of :python:`'conditionalJumpTrigger0, conditionalJumpTrigger1, conditionalJumpTrigger2'` to the set attribute function.
 
 
+sites
+-----
+
+    .. py:attribute:: nidigital.Session.sites[]
+
+        If no prefix is added to the items in the parameter, the correct prefix will be added when
+        the driver function call is made.
```

Comment:
Do you want me to change this in this PR? It is unrelated since that is boilerplate text that applies to all repeated capabilities?

Moved to #1334